workshops/drawing_shapes_workshop/README.md

```diff
@@ -0,0 +1,309 @@
+
+<img src="https://cloud-mo50706v1.vercel.app/0screenshot__1394_.png" width="380" alt="Triangle Example">
+
+```python
+def triangle(x, y):
+  turtle.goto(x-(triangleLength/2), y-((triangleLength *math.sqrt(3))/6))
+  turtle.setheading(0)
+
+  turtle.begin_fill()
+  for x in range(3):
+    turtle.forward(triangleLength)
+    turtle.left(120) 
+  turtle.end_fill()
+
+  turtle.setheading(0)
+```
+- The def defines the function, and we are passing in x and y coordinates as parameters.
+- We are going to the offset coordinate and setting the direction to face right.
+- We are beginning the fill and creating a for loop.
+- We are going forward and then turning left 3 times in order to create the triangle.
+- We finally end the fill and set the direction to the right.
+
+
+# Finishing Up the Shapes
+
+<img src="https://media1.tenor.com/images/46422729dc9ecd6898203ef13b3d9985/tenor.gif?itemid=15700153" width="380" alt="Almost There Gif">
+
+## Switching Between Shapes
+Here is the functionality to switch between shapes.
+
+```python
+def switchShape(x, y):
+  global itemNum
+  global maxItem
+
+  itemNum += 1
+
+  if (itemNum >= maxItem):
+    itemNum = 0
+```
+- The itemNum and maxItem are made global so we can use them.
+- Increment the itemNum and if greater than or equal to the maxItem amount, go back to 0.
+- Even though we are not using the parameters, they are required for our screen click function later.
+
+## Drawing Shapes
+Here is the functionality to draw the current shape.
+
+```python
+def drawItem(x, y): 
+  global itemNum
+  if (itemNum == 0):
+    circle(x, y)
+  elif (itemNum == 1):
+    square(x, y)
+  elif (itemNum == 2):
+    triangle(x, y)
+  else:
+    print(itemNum)
+```
+- The "itemNum" variable is made global so we can use it.
+- We then see what value it is and draw the respective shape.
+
+# Switching Between Colors
+We are changing the color.
+
+<img src="https://cloud-c9ojk6h90.vercel.app/0screenshot__1395_.png" width="380" alt="Switch Colors Example">
+
+```python
+def switchColor(x, y):
+  global colorNum
+  global colors
+
+  colorNum += 1
+
+  if (colorNum >= len(colors)):
+    colorNum = 0
+    
+
+  turtle.color(colors[colorNum])
+```
+- The "colorNum" and "colors" variables are made global so we can use them.
+- Increment the "itemNum" and if greater than or equal to the "colors" length, go back to 0.
+- We then set the turtle's color to the color at the colorNum index of the colors array.
+
+
+# Clicking On Screen
+
+<img src="https://media3.giphy.com/media/hCfESQ8r1eBOg/giphy.gif" width="380" alt="Clicking Screen Gif">
+
+```python
+#everything we already wrote
+turtle.onscreenclick(drawItem, 1)
+turtle.onscreenclick(switchColor, 2)
+turtle.onscreenclick(switchShape, 3)
+```
+- The "turtle.onscreenclick()" function takes in a function with x and y parameters and the mouse button.
+- If we click the left button (1), draw the current shape.
+- If we click the middle button (2), switch the color.
```

Comment:
fyi this doesn't work on MacBooks

Reply:
Oh man didn't even think of this. Not really sure how to handle it. Any ideas?

Reply:
is there a way to double click?

Reply:
No but I can try to see if I can maybe get keyboard input to work as a secondary option.

Reply:
cool! thanks Jake